### DISCUSSION TITLE
請問目前應該用什麼版本漢化?

### DISCUSSION THREAD

**Discussion by sosad2036** [GpointChen/FFXIVChnTextPatch-GP#196] [Q&A]
我目前下載了 FFXIVChnTextPatch-0706 , 也漢化完畢 , 但進遊戲都是英文

我是不是下載錯誤版本?

**Comment by GpointChen** [comment]
1. 你在設定中是選擇覆蓋英文嗎？
2. 全部地方都是英文嗎？主畫面和大廳是不會漢化的
